platooning_strategic/include/platoon_strategic_plugin_node.h

```diff
@@ -32,7 +32,7 @@
 namespace platoon_strategic
 {
 /**
- * \brief ROS node for the YieldPlugin
+ * \brief ROS node for the yield_plugin
```

Comment:
I guess the yield plugin was used as a template for this? Should be platoon_strategic I believe.